proto/types/commonpb/v3/common.proto

```diff
@@ -289,17 +289,17 @@ message QueryOptions {
 
 // ParalusConditionStatus is the status of the status condition
 enum ParalusConditionStatus {
-  NotSet = 0;
-  Pending = 1;
-  InProgress = 2;
-  Success = 3;
-  Failed = 4;
-  Retry = 5;
-  Skipped = 6;
-  Stopped = 7;
-  Expired = 8;
-  Stopping = 9;
-  Submitted = 10;
+  PARALUS_CONDITION_STATUS_NOT_SET_UNSPECIFIED = 0;
```

Comment:
From a quick looks, it looks like most of the `0` values are set as `NOT_SET`, let us change the lint rule to use `NOT_SET` instead of `UNSPECIFIED`.